content/en/docs/reference/using-api/cel.md

```diff
@@ -0,0 +1,330 @@
+| `names.isSorted()`                                                                 | Verify that a list of names is kept in alphabetical order |
+| `items.map(x, x.weight).sum() == 1.0`                                              | Verify that the "weights" of a list of objects sum to 1.0 |
+| `lowPriorities.map(x, x.priority).max() < highPriorities.map(x, x.priority).min()` | Verify that two sets of priorities do not overlap         |
+| `names.indexOf('should-be-first') == 1`                                            | Require that the first name in a list if a specific value | 
+
+See the [Kubernetes List Library](https://pkg.go.dev/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/cel/library#Lists)
+godoc for more information.
+
+### Kubernetes Regex Library
+
+In addition to the `matches` function provided by the CEL standard library, the
+regex library provides `find` and `findAll`, enabling a much wider range of
+regex operations.
+
+Examples:
+
+| CEL Expression                                              | Purpose                                                  |
+|-------------------------------------------------------------|----------------------------------------------------------|
+| `"abc 123".find('[0-9]*')`                                  | Find the first number in a string                        |
+| `"1, 2, 3, 4".findAll('[0-9]*').map(x, int(x)).sum() < 100` | Verify that the numbers in a string sum to less than 100 |
+
+See the [Kubernetes Regex Library](https://pkg.go.dev/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/cel/library#Regex)
+godoc for more information.
+
+### Kubernetes URL Library
+
+To make it easier and safer to process URLs, the following functions have been added:
+
+- `isURL(string)` - Check if a string is a valid URL according to the [Go's
+  net/url](https://pkg.go.dev/net/url#URL) package. The string must be an
+  absolute URL.
+- `url(string) URL` Converts a string to a URL or results in an error if the
+  string is not a valid URL.
+
+Once parsed via the `url` function, the resulting URL object has `getScheme`,
+`getHost`, `getHostname`, `getPort`, `getEscapedPath` and `getQuery` accessors.
+
+Examples:
+
+| CEL Expression                                                  | Purpose                                        |
+|-----------------------------------------------------------------|------------------------------------------------|
+| `url('https://example.com:80/').getHost()`                      | Get the 'example.com:80' host part of the URL. |
+| `url('https://example.com/path with spaces/').getEscapedPath()` | Returns '/path%20with%20spaces/'               |
+
+See the [Kubernetes URL Library](https://pkg.go.dev/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/cel/library#URLs)
+godoc for more information.
+
+### Kubernetes Authorization Library
+
```

Comment:
Note: This is going to be available starting 1.27 right? I noticed this pr is against `main` which means it will be available as soon as merged. 
I am ok with it since it mentioned checking API doc for `authorizer` variable just a note.

Reply:
Since this doesn't reference available particular capabilities in particular APIs, I don't see this as being particularly version sensitive. See below guidance about merging to main first.